wasmedge/ast.go

```diff
@@ -60,6 +60,7 @@ func (self *AST) ListImports() []*ImportType {
 			C.WasmEdge_ASTModuleListImports(self._inner, &(cimptype[0]), ltypes)
 		}
 		for i, val := range cimptype {
+			imptype[i] = &ImportType{}
```

Comment:
I think move the whole for loop into the if statement is better?

```go
if uint(ltypes) > 0 {
	imptype = make([]*ImportType, uint(ltypes))
	cimptype = make([]*C.WasmEdge_ImportTypeContext, uint(ltypes))
	C.WasmEdge_ASTModuleListImports(self._inner, &(cimptype[0]), ltypes)
	for i, val := range cimptype {
		imptype[i]._inner = val
		imptype[i]._ast = self._inner
		imptype[i]._own = false
	}
	return imptype
} else {
	return nil
}
```

Reply:
done in https://github.com/second-state/WasmEdge-go/pull/38/commits/38caaf64a4b7c60cce3c825074232e124b9141a0